.github/workflows/step-run-tests.yml

```
@@ -123,21 +129,56 @@
       timeout-minutes: 10
 
     - name: Запуск тестирования (Windows)
       if: runner.os == 'Windows'
       shell: pwsh
       run: |
-        Start-Process ibsrv.exe -ArgumentList "--db-path=file-db"
-        Start-Process -NoNewWindow -PassThru -Wait 1cv8c.exe -ArgumentList '/WS "http://localhost:8314" /C"RunUnitTests=yaxunit-config.json" /L ${{ steps.short_locale.outputs.substring }} /VL ${{ inputs.locale }} /DisableStartupDialogs /DisableStartupMessages /DisableUnrecoverableErrorMessage /Out 1cv8c-output.log'
+        # Запуск ibsrv в фоне
+        Start-Process ibsrv.exe -ArgumentList "--db-path=file-db" -WindowStyle Hidden
+        Start-Sleep -Seconds 5
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)